content/00.build/40.tooling/20.hardhat/01.guides/10.getting-started.md

```diff
@@ -60,29 +30,31 @@ This method allows you to test contracts without requesting external testnet fun
 - Contracts compiled with other compilers will fail to deploy to ZKsync Era.
 ::
 
-## Project setup
-
-To create a new project run the `zksync-cli create` command, passing a project name:
-
-#### Solidity project
-
-```sh
-npx zksync-cli create demo --template hardhat_solidity
-```
+## ZKsync Hardhat Plugins
 
-#### Vyper project
-
-```sh
-npx zksync-cli create demo --template hardhat_vyper
-```
+ZKsync Era has the following official plugins for Hardhat:
 
-This command creates a `demo` folder and clones a Hardhat template project inside it.
-The downloaded project is already configured and contains all the required plugins.
+- [@matterlabs/hardhat-zksync](hardhat-zksync) - used to access to all of the supported plugins and to use them
+as needed in your project. This should be the primary plugin most developers will need to use.
+- [@matterlabs/hardhat-zksync-solc](hardhat-zksync-solc) - used to compile contracts written in Solidity.
+- [@matterlabs/hardhat-zksync-vyper](hardhat-zksync-vyper) - used to compile contracts written in Vyper.
+- [@matterlabs/hardhat-zksync-deploy](hardhat-zksync-deploy) - used to deploy smart contracts.
+- [@matterlabs/hardhat-zksync-verify](hardhat-zksync-verify) - used to verify smart contracts.
+- [@matterlabs/hardhat-zksync-verify-vyper](hardhat-zksync-verify-vyper) - used to verify vyper smart contracts.
+- [@matterlabs/hardhat-zksync-upgradable](hardhat-zksync-upgradable) - used to deploy, update, and verify proxy smart contracts.
+- [@matterlabs/hardhat-zksync-ethers](hardhat-zksync-ethers) - wrapper around zksync-ethers with some extra Hardhat-specific functionality.
+- [@matterlabs/hardhat-zksync-node](hardhat-zksync-node) - used to run the ZKsync era-test-node locally.
 
-::callout
+::callout{icon="i-heroicons-light-bulb"}
 If you want to migrate an existing project, please check the [project migration guide](migrating-to-zksync).
```

Comment:
Since you are moving the page for Migrating to ZKsync, this URL needs to be updated.